app/sidekiq/form526_submitted_email_job.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+require 'va_notify/service'
+
+class Form526SubmittedEmailJob
+  include Sidekiq::Job
+  include SentryLogging
+  sidekiq_options expires_in: 1.day
+
```

Comment:
This will fall back on the default retried of 25. is that okay?

Reply:
@rmtolmach this is the same configuration as the previous Form526ConfirmationEmail - i didn't change anything between the two 🐱 
so i'm guessing it's ok